### ISSUE TITLE
possible simplification in checking parallel

### ISSUE THREAD

**Issue by jakejh** [hugheylab/limorhyde2#18]
Is there a reason to use `sys.status()` instead of just calling `sys.calls()` directly?

https://github.com/hugheylab/limorhyde2/blob/a2da8d8dafa29a6bcbfdeef7e771160580f7bd3e/R/get_stats_utils.R#L156-L160

**Comment by JSchoenbachler** [comment]
Good call, I forgot I could call that. Do you want me to make a PR or just push directly? I already have a local change and have verified that this works.

**Comment by JSchoenbachler** [comment]
Closed with https://github.com/hugheylab/limorhyde2/pull/19
